src/test/java/guitests/FilterTextFieldTest.java

```diff
@@ -121,6 +127,20 @@ public void cancelTest() {
         awaitCondition(() -> toggle.get() % 2 == 1);
     }
 
+    @Test
+    public void helpTest() {
```

Comment:
OK. I followed the local naming conventions from the other tests in this file, so I will change them to verb form as well.
Should I spawn a new issue for the name refactoring, or should I just do it in this PR?